src/main/java/io/reactivex/Flowable.java

```diff
@@ -3757,25 +3757,29 @@ public static int bufferSize() {
      * the parameter name.
      * @param value the value to validate
      * @param paramName the parameter name of the value
+     * @return the value
      * @throws IllegalArgumentException if bufferSize &lt;= 0
      */
-    private static void verifyPositive(int value, String paramName) {
+    protected static int verifyPositive(int value, String paramName) {
```

Comment:
These aren't high-value APIs and are trivial to replicate with
```java
if (value < 0) throw new IllegalArgumentException("value < 0: " + value);
```
I would prefer they be moved somewhere in `internal` and made public for use all over the library but not be exposed in the library's public API.